### PR TITLE
fix: install-egctl doc dead link

### PR DESCRIPTION
**Pull request by yeedove**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

- https://gateway.envoyproxy.io/v1.0.0/install/install-egctl/
- fix install-egctl doc dead link

### CODE REVIEW

File: site/content/en/latest/install/install-egctl.md [hunk 1]
```diff
@@ -52,6 +52,6 @@ curl -fsSL https://gateway.envoyproxy.io/get-egctl.sh | VERSION=latest bash
 
 {{% alert title="Next Steps" color="warning" %}}
 
-You can refer to [User Guides](/latest/user/egctl) to more details about egctl.
```
[the remark this answers is not in the document]
Comment (yeedove) [reply site/content/en/latest/install/install-egctl.md:55]: yes, done！

**Review by arkodg — DISMISSED** [review]
LGTM thanks !

File: site/content/en/v0.6.0/install/install-egctl.md [hunk 2]
```diff
@@ -52,6 +52,6 @@ curl -fsSL https://gateway.envoyproxy.io/get-egctl.sh | VERSION=latest bash
 
 {{% alert title="Next Steps" color="warning" %}}
 
-You can refer to [User Guides](/latest/user/egctl) to more details about egctl.
+You can refer to [User Guides](/latest/user/operations/egctl) to more details about egctl.
```
Comment (arkodg) [inline site/content/en/v0.6.0/install/install-egctl.md:55]:
```suggestion
You can refer to [User Guides](../user/operations/egctl) to more details about egctl.
```

File: site/content/en/v1.0.0/install/install-egctl.md [hunk 3]
```diff
@@ -52,6 +52,6 @@ curl -fsSL https://gateway.envoyproxy.io/get-egctl.sh | VERSION=latest bash
 
 {{% alert title="Next Steps" color="warning" %}}
 
-You can refer to [User Guides](/latest/user/egctl) to more details about egctl.
+You can refer to [User Guides](/latest/user/operations/egctl) to more details about egctl.
```
Comment (arkodg) [inline site/content/en/v1.0.0/install/install-egctl.md:55]:
```suggestion
You can refer to [User Guides](../user/operations/egctl) to more details about egctl.
```

File: site/content/en/latest/install/install-egctl.md [hunk 4]
```diff
@@ -52,6 +52,6 @@ curl -fsSL https://gateway.envoyproxy.io/get-egctl.sh | VERSION=latest bash
...
+You can refer to [User Guides](/latest/user/operations/egctl) to more details about egctl.
```
Comment (arkodg) [inline site/content/en/latest/install/install-egctl.md:55]:
```suggestion
You can refer to [User Guides](../user/operations/egctl) to more details about egctl.
```

**Comment by arkodg** [comment]
we need to link to version specific pages, added some comments around that

File: site/content/en/v0.6.0/install/install-egctl.md (hunk 2 above)
Comment (yeedove) [reply site/content/en/v0.6.0/install/install-egctl.md:55]: It does not exist in the v.0.6.0 doc. Do we need to link it to the latest url or `../user/egctl` ? i personally prefer the latter.

**Comment by zirain** [comment]
once agin, we really need preview -3-

**Review by arkodg — APPROVED** [review]

**Review by zirain — APPROVED** [review]

**Review by arkodg — APPROVED** [review]

**Pull request merged**